### DISCUSSION TITLE
피움 프로젝트 앞으로의 진행방향

### DISCUSSION THREAD

**Discussion by Choi-JJunho** [woowacourse-teams/2023-pium#473] [뜨거운 토론]
# 서론

우아한테크코스 레벨 4를 마무리하며 피움 프로젝트 유지방안에 대해 이야기를 나눠보려고합니다.

앞으로의 프로젝트의 방향성과 참여인원에 대한 이야기가 주가 될 것 같습니다.

# 본론

## 참여 인원

- 주노: 학교로 돌아가서 무조건 해야합니다ㅠㅠ
- 참새: 저는 불법백수동안 할 의향있습니다~
- 클린: 상동입니다. 여유롭게 일정을 가져가면 좋을 것 같아요.
- 조이: 상동. 저도 여유롭게 가져가면 좋겠슴다.
- 그레이: 동일합니다. 근데 다음학기에 전공 4과목 남아서 흑흑 취업하면 안해도되는데 현실이….
- 하마드: 취준기간 동안은 참여가 힘들 것 같아요 근데 또 취준기간이 길어지면 모르겠네요 올해는 힘들 것 같아요 돌아오고는 싶은데 그때까지 피움이 살아있을진ㅠㅠ
- 쵸파: 클 참과 동일해요 아… 불법백수가 되느냐 아니냐가 중요한데…

## 피움 프로젝트 유지 방안 (회의록)

- 얼마나 유지할건지
    - 모두가 취업할 때 까지?
- 신규 기능을 지속적으로 개발할 것인지?
    - 나중에 상의해야 할 문제
    - 식물 종류 문제
        - 마지막 데모데이에 로이가 식물 많다고 했던게 유일한 칭찬 ㅠㅠ
    - 이거보다는 사용자 유치가 좀더 급할지도
    - 기능이 아니더라도 코드 리팩터링으로 들어가도 ㄱㅊ
- 서버 개인 환경으로 이관 시 서버비 부담
    - 우테코에서 지원해주는 기간 확인
        - 레벨1?2?까지는 유지를 해주긴 한대요
        - 그 뒤에 이관
    - 프리티어 1년씩 돌아가면서 사용하기 (1년마다 서버 재구축해야하긴함)
    - 팀 전체 N빵 or 프로젝트 가져가는 사람들끼리 부담
    - 처음 취업하는 사람이 내기 ㅎㅎ ^-^
        - 1년치 시원~하게 긁죠?
    - 어쩄든 우테코가 유지시켜주니까 그거 끝날때쯤 정합시다
- 사용자 유치?
    - 플젝을 신나게 유지하려면 사용자가 필요하긴 해요
    - 잠실캠 식물에 물주시는분들이 식물 하나하나마다 물을 주고 팻말을 꽂으시더라구요 그분들의 피드백을 받는 것도 좋을 듯 합니다

# 결론

- 수료하고 나서 다시 이야기하자!

**Comment by WaiNaat** [comment]
좋은데요?

**Comment by yeonkkk** [comment]
좋은데요?
